docs/emitters/json-schema/reference/emitter.md

```diff
@@ -21,6 +21,15 @@ emit:
   - "@typespec/json-schema"
 ```
 
+The config can be extended with options as follows:
+
+```yaml
+emit:
+  - "@typespec/json-schema"
+options:
+  "@typespec/json-schema": option:value
```

Comment:
I think it's missing a new line and indentation here

Reply:
I'm struggling to get it formatted correctly -- any suggestions? This is how it looks in code, which *should* in theory be right

![image](https://github.com/user-attachments/assets/10e8871a-c6d2-46ba-bcfe-d62bb5619603)

Reply:
Thanks to Timothee for the fix! Was missing a space between "option:" and "value"